### PR TITLE
avoid _define_scope_methods! conflicts when activerecord and mongoid required at the same time

### PR DESCRIPTION
**Pull request by matsu911**

### CODE REVIEW

File: lib/enumerize/scope/activerecord.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ def enumerize(name, options={})
         _enumerize_module.dependent_eval do
           if self < ::ActiveRecord::Base
             if options[:scope]
-              _define_scope_methods!(name, options)
+              _define_acriverecord_scope_methods!(name, options)
```
Comment (lest) [inline lib/enumerize/scope/activerecord.rb]: Can you please fix acriverecord -> activerecord?

Comment (matsu911) [inline lib/enumerize/scope/activerecord.rb]: sorry, fixed

File: test/test_helper.rb [hunk 2]
```diff
@@ -3,6 +3,7 @@
 require 'active_support/core_ext/kernel/reporting'
 require 'active_model'
 require 'rails'
+require 'mongoid'
```
Comment (lest) [inline test/test_helper.rb]: This `require` causes mongomapper tests failure on Travis. Can you please wrap it in `begin ... rescue LoadError`?

**Comment by matsu911** [comment]
should i rebase them?

File: test/test_helper.rb [hunk 3]
```diff
@@ -3,6 +3,11 @@
 require 'active_support/core_ext/kernel/reporting'
 require 'active_model'
 require 'rails'
+begin
+  require 'mongoid'
+rescue LoadError => e
+  puts e
```
Comment (lest) [inline test/test_helper.rb]: I think there's no need to output the exception, so we can change it to:

```ruby
begin
  require 'mongoid'
rescue LoadError
end
```

Can you please apply this change? Thanks!

**Comment by lest** [comment]
Thanks!

**Pull request merged**